src/js/components/lazy-load.js

```diff
@@ -107,20 +188,37 @@ class LazyLoad {
     return image
   }
 
-  destroy () {
-    $(window).off('scroll', this.onScrollHandler)
+  /**
+   * Parse string with breakpoint size and image sorce
+   * convert each breakpoint into an object.
+   *
+   * @param  {[String]} breakpoints [breakpoints separated by comma]
+   * @return {[Array]}             [description]
+   */
+  parseBreakpoints (breakpoints) {
+    return breakpoints.split(/,\s+/g).map(breakpoint => {
+      const [ src, width ] = breakpoint.trim().split(/\s+/)
+
+      return { src, width }
+    })
   }
-}
 
-/* istanbul ignore next */
-$.fn[NAME] = function (options) {
-  options = options || {}
+  /**
+   * Sort breakpoint array to greater to lower
```

Comment:
`Sort breakpoints array from greater to lower values.`
I'm adding periods (`.`) to the end of every sentence so if you think we should make that a standard, that would be cool :nail_care: